src/core/CCFormat.mli

```diff
@@ -55,7 +56,7 @@ val flush : unit printer
     @since 1.2 *)
 
 val string_quoted : string printer
-(** Similar to {!CCString.print}.
+(** Like {!CCString.print}.
```

Comment:
what is the point of this change? :/

Reply:
Simple homogeneity.
The goal is to replace ```'Similar to {...' or 'Analog to {...'``` with ```'Like {...'.```

Reply:
Honestly, this seems like a lot of noise for nothing. I don't care about homogeneity of wording in comments…

Reply:
Replacing some identical expressions with a simple one "seems like a lot of noise for nothing".

Very strange reflexion! 

But, anyway... Reverting.